.pre-commit-config.yaml

```diff
@@ -1,17 +1,17 @@
 exclude: ^(.github|.changes|docs/|boto3/compat.py|boto3/data|CHANGELOG.rst)
 repos:
   - repo: 'https://github.com/pre-commit/pre-commit-hooks'
-    rev: v4.2.0
+    rev: v4.3.0
     hooks:
       - id: check-yaml
       - id: end-of-file-fixer
       - id: trailing-whitespace
   - repo: 'https://github.com/asottile/pyupgrade'
-    rev: v2.32.1
+    rev: v2.34.0
     hooks:
       - id: pyupgrade
         args:
-          - '--py36-plus'
+          - '--py37-plus'
```

Comment:
We left this unchanged intentionally for the moment. We’ll look at making the breaking changes once a few other pieces are unblocked.